### ISSUE TITLE
0.5.18: Windows: Starting up the program and checking if it connects with tor enabled and then with tcp enabled is not working

### ISSUE THREAD

**Issue by Tas4tari** [tari-project/universe#819]
**Describe the bug**
As a user I find when I install the app, disable Tor, close the app, delete the node file and restart, I get stuck at `connecting network peers` for the past 35+ minutes. 
**Additional info:**
I checked on com.tari.universe and the node file had been addded again

**To Reproduce**
Test Starting up the program and checking if it connects with tor enabled and then with tcp enabled:

1. User should have Tari Universe installed on their computer
2. User should have Tari Univers open
3. Click on the Settings icon
4. Go to Experimental Settings
5. Scroll down to the Tor toggle
6. User should see the Use Tor toggle on by default
7. User should be able to toggle the Use Tor toggle on/off
8. When Tor is toggled off, confirm the app asks the user to restart
9. User should say no and close the app
10. Go to com.tari.universe and delete the node data folder: Windows: AppData\Local\node & Mac: Application Support\node
11. User should restart the app
12. Confirm the user is connected to the Tari network and the chain resyncs

**Expected behavior**
On restart of the app, user should be be connected to the Tari network, the chain resynces and the user should be able to mine.

**Screenshots**
![Screenshot (53)](https://github.com/user-attachments/assets/219bdb91-6f04-4c33-929b-add59d1abb32)

**Desktop (please complete the following information):**
 - Aspire A315-59
 - Windows 11, version 23H2

**Comment by Tas4tari** [comment]
This is still an issue on 0.5.22

**Comment by Tas4tari** [comment]
This is still an issue on 0.5.24

**Comment by Tas4tari** [comment]
Tested on NextNet: version: 0.5.25 and this is still an issue

**Comment by Tas4tari** [comment]
Tested on NextNet version 0.5.27 and this is still an issue

**Comment by Tas4tari** [comment]
Tested on v0.5.28 and issue is still prevalent

**Comment by Anziskavds** [comment]
When a user toggled Tor off and clicked cancel on the restart pop up and manually restarted the miner, on start up the user received a critical error pop up message. I clicked on the close Tari universe button and tried to open Tari Universe again but I got stuck on 43%. I needed to force quite Tari Universe and set Tor to true on visual studio to get Tari Universe up and running again. This is on a MacBook Air 13 inch M1 2020 on Testnet v0.5.28

**Comment by mmrrnn** [comment]
Related to #874

**Comment by Anziskavds** [comment]
This issue is resolved on Testnet 0.5.41 on a MacBook Air 13 inch M1 2020.

**Comment by mmrrnn** [comment]
"The problem is that we've got few tcp peers. You tried to connect when there wasn't tcp peers at all. It means you really hasn't been connected to the Tari network"

**Comment by Anziskavds** [comment]
Issue prevalent on 0.5.42 again on a MacBook Pro 13 inch M1 2020 Sonoma 14.5

**Comment by Anziskavds** [comment]
Issue is resolved on Testnet v0.5.45.

**Comment by Tas4tari** [comment]
Still prevalent on Windows 11, app version 0.5.56

**Comment by Tas4tari** [comment]
Tested on App version 0.5.58, Windows 11 and this is still prevalent

**Comment by Tas4tari** [comment]
Tested on WIndows 11 on app version 0.5.60 and this is still an issue

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.5.61 and this is still aan issue

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.6.4 and this is still an issue as I am now stuck on 43%:
![Screenshot (230)](https://github.com/user-attachments/assets/232a2028-5939-4267-a522-f858418013cf)

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.6.6 and this is still an issue as I am now stuck on 43%:

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.6.15 and this is still an issue as I am now stuck on 43%:
![Screenshot (283)](https://github.com/user-attachments/assets/a6582602-1010-4097-9c2c-71ce744e6785)

**Comment by mmrrnn** [comment]
@brianp hard fork issue?

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.6.16 and this is still an issue where I am still stuck at 43%

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.7.0 and this is still an issue where I am still stuck at 43%

**Comment by brianp** [comment]
Marking this as on hold until we make more dns/peer adjustments. 

I've discussed the changes with @leet4tari, and then I got busy with other things. So we'll come back around to this eventually. I'd say there's no need to keep testing it for now. Keep it as a known issue but problem lays outside of universe itself.

**Comment by Tas4tari** [comment]
User is stuck @ 43% initialization on Windows 11, app version 0.7.7

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.8.2 and issue is still occurring:
2 scenarios checked:

Switch Tor of and restart app:
User sees connection displayed as disconnected from the Tari network
Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Will do so.

**Comment by Tas4tari** [comment]
Tested on Windows 11, and this is still an issue for me on app version 0.8.5

**Comment by Tas4tari** [comment]
When Tor is disabled, network appears as disconnected on Windows 11, app version 0.8.5

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.8.7 and issue is still occurring:
2 scenarios checked:

Switch Tor of and restart app:
User sees connection displayed as disconnected from the Tari network
Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.8.7 and issue is still occurring:

- Switch Tor of and restart app:

User sees connection displayed as disconnected from the Tari network
![Screenshot (455)](https://github.com/user-attachments/assets/c1352eb0-7b6f-4a14-b553-501914868d49)
![Screenshot (456)](https://github.com/user-attachments/assets/c8d03f3e-19de-444d-abcf-da4c91fe951f)

**Comment by Tas4tari** [comment]
Tested on Windows 11

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app

App is restarted, however user is disconnected from the network
![Screenshot (459)](https://github.com/user-attachments/assets/14a70498-761b-4060-aa5f-1eb3c8df8cb2)

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.8.9 and issue is still occurring:

Switch Tor of and restart app:
User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.13 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - App restarted, but was disconnected from the Tari Network

**Comment by Tas4tari** [comment]
Tested on windows 11, app version 0.8.13 and issue is still prevalent:

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.8.14

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - App is restarted, however user is disconnected from the network. And user sees hard fork notification

![Screenshot (500)](https://github.com/user-attachments/assets/250efeee-e8cd-4b08-a7ca-35ff44cc8b8c)

**Comment by Tas4tari** [comment]
Tested on windows 11, app version 0.8.13 and issue is still prevalent:

- Switch Tor off and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.25 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.27 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.27 and issue is still prevalent:

- When Tor is disabled, network appears as disconnected on Windows 11, app version 0.8.25

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.30 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.30 and issue is still prevalent:

- When Tor is disabled, network appears as disconnected on Windows 11, app version 0.8.25

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.34 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.34 and issue is still prevalent:

- When Tor is disabled, network appears as disconnected on Windows 11, app version 0.8.34

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.37 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 43% on initializing screen

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.38 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% on initializing screen
![Screenshot (31)](https://github.com/user-attachments/assets/3cae4382-68e0-411b-a269-b6b46958cef1)

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.8.38 and this is still occurring
Switch Tor of and restart app:
User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.40 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% on initializing screen
![Screenshot (38)](https://github.com/user-attachments/assets/ec3c08b1-c12b-46c3-a77b-72c6c092f260)

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.40 and issue is still prevalent:

- When Tor is disabled, network appears as disconnected on Windows 11, app version 0.8.40

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.41 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - App starts however it shows as disconnected from the Tari Network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.42 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.8.44 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.45 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.8.45 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.49 and issue is still prevalent:

Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.8.50 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.8.50 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.1 and issue is still prevalent:

Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested successfully on Windows, app version 0.9.2:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - I was able to load the app and it was connected to the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.3 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

![Image](https://github.com/user-attachments/assets/ee2c114a-cf73-4fab-b4a1-73a61570c148)

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.4 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.5 and issue is still prevalent:

Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.5 and this is still occurring

Switch Tor of and restart app:
User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.6 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.8 and issue is still prevalent:
- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.801 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.801 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.802 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.803 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.805 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.805 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.807 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.809 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.809 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.810 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
- Tested in Windows 11, app version 0.9.811 and this is still occurring
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.812 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.812 and this is still occurring

- Switch Tor of and restart app:
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.813 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.816 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.816 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.817 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 48% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.817 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.818 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.819 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.819 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.820 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.823 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.823 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.825 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.826 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.826 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.834 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.837 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.837 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.838 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.838 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.840 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.840 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested on Windows, app version 0.9.841 and issue is still prevalent:

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.841 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.846 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.846 and this is still occurring

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.850 and this is still occurring

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 61% initialization

![Image](https://github.com/user-attachments/assets/95c122ca-ab9b-45fc-a1cb-faf5d844495a)

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.850 and this is still occurring

- Switch Tor of and restart app:
  - User sees connection displayed as disconnected from the Tari network

![Image](https://github.com/user-attachments/assets/8c7c0adc-9c85-465a-bf1b-bf6cb7da5258)

**Comment by Tas4tari** [comment]
Tested in Windows 11, app version 0.9.906 and this is still occurring

Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
User is stuck at 37% initialization

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.9.910 and this is still occurring

- Switch Tor off, do not restart, close app and delete node folder from App Data > Local > com.tari.universe & restart the app
  - User is stuck at 33% `Setting up the Tari Node | Connecting to network peers 0/3`. I had to enable tor to true in the config and then restart for the app to load completely

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 0.9.913, I get stuck at `Setting up the Tari Node | Connecting to network peers 0/3`
- Setting Tor to true in config allows me to start up the app without issue


![Image](https://github.com/user-attachments/assets/d17f3d95-525f-4067-b3bb-b8390827bd40)

Saw the ffg error in logs:
`2025-05-01 18:35:11.505362700 ERROR Failed to send websocket message: Trying to work with closed connection // src\websocket_manager.rs:343`


[universe.log](https://github.com/user-attachments/files/20002161/universe.log)

**Comment by Tas4tari** [comment]
On Windows 11, app version 0.9.914, I find that I am stuck on 33%
`Setting up the Tari Node | Connecting to network peers 0/3`

![Image](https://github.com/user-attachments/assets/6f56c924-9342-44b1-9012-c31a70a8938b)

When I check the logs, I see the following errors for minotari:
`2025-05-05 10:46:11.188098900 INFO  Node is not synced, skipping orphan chain check // src\node\node_adapter.rs:314
2025-05-05 10:46:16.840863900 ERROR local_minotari_node is not healthy. Health check returned warning // src\process_watcher.rs:276
2025-05-05 10:46:16.950275300 INFO  Stopping local_minotari_node process with exit code: 0 // src\process_adapter.rs:222
2025-05-05 10:46:16.950952000 INFO  local_minotari_node exited successfully // src\process_watcher.rs:315
2025-05-05 10:46:17.956186600 WARN  Restarting local_minotari_node after health check failure // src\process_watcher.rs:325`


[universe.log](https://github.com/user-attachments/files/20033727/universe.log)

[logs.zip](https://github.com/user-attachments/files/20033768/logs.zip)

**Comment by Tas4tari** [comment]
Tested on app version 1.0.0 on Windows 11 and this is still an issue:
When app is restarted, I get stuck at 27% and have to enable Tor via app config and then restart in order for the app to start successfully

**Comment by Tas4tari** [comment]
Tested on app version 1.0.5 on Windows 11 and this is still an issue:
When app is restarted, I get stuck at 23% and have to enable Tor via app config and then restart in order for the app to start successfully


![Image](https://github.com/user-attachments/assets/234e51f9-fa64-4e03-949e-376dd6e787c5)

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.0.10 and user is stuck at 33% of the initialization phase

![Image](https://github.com/user-attachments/assets/ee50f0f0-ab10-46f3-8f5d-0dec72797ec5)

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.2.9
I find when I switch Tor off and delete the node file; when I restart the app, I get stick at `Connecting network peers 0/3`
Only after I close the app, go to app config and enable Tor, then restart am I able to move forward

![Image](https://github.com/user-attachments/assets/63650e58-8c7b-4644-abb9-3c843b558486)

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.2.12
I find when I switch Tor off and delete the node file; when I restart the app, I get stick at Connecting network peers 0/3
Only after I close the app, go to app config and enable Tor, then restart am I able to move forward

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.2.13
I find when I switch Tor off and delete the node file; when I restart the app, I get stick at Connecting network peers 0/3
Only after I close the app, go to app config and enable Tor, then restart am I able to move forward

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.2.15
I find when I switch Tor off and delete the node file; when I restart the app, I get stick at Connecting network peers 0/3
Only after I close the app, go to app config and enable Tor, then restart am I able to move forward

![Image](https://github.com/user-attachments/assets/b9600ec7-8e97-4e3d-8663-8edae78b919f)

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.0
I find when I switch Tor off and delete the node file; when I restart the app, I get stick at Connecting network peers 0/3
Only after I close the app, go to app config and enable Tor, then restart am I able to move forward

<img width="1270" height="755" alt="Image" src="https://github.com/user-attachments/assets/b8fe42bb-806b-422c-958d-8143f6ef7bd7" />

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.2
I find when I switch Tor off and delete the node file; when I restart the app, I get stick at Connecting network peers 0/3 for `Local node` however I am connected via `Remote`
To get connected to `Local`. I need to close the app, enable Tor via app config and then restart

<img width="1245" height="750" alt="Image" src="https://github.com/user-attachments/assets/c8291969-c129-429a-aaa8-b8dd4377655c" />

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.4
- I find when I switch Tor off and delete the node file; when I restart the app, I get stick at `Connecting network peers 0/3` for Local node however I am connected via Remote
To get connected to Local. I need to enable Tor and restart the app 


<img width="1261" height="755" alt="Image" src="https://github.com/user-attachments/assets/b31b7452-a59b-45c9-b47a-6c2b3a6263b7" />
After enabling Tor and restarting:


<img width="1604" height="949" alt="Image" src="https://github.com/user-attachments/assets/a5252d41-3420-488c-a44d-6cbf4ad3d17c" />

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.6

- I find when I switch Tor off and delete the node file; when I restart the app, I get stick at Connecting network peers 0/3 for Local node however I am connected via Remote
  - To get connected to Local. I need to enable Tor and restart the app

<img width="985" height="531" alt="Image" src="https://github.com/user-attachments/assets/76ab257d-c8c5-47ea-8f50-902eaecdc029" />

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.7
I find when I switch Tor off and delete the node file; when I restart the app, I get stuck at Connecting network peers 0/3 for Local node however I am connected via Remote
To get connected to Local. I need to enable Tor and restart the app

**Screenshot:**
<img width="926" height="209" alt="Image" src="https://github.com/user-attachments/assets/2685a15d-8a94-4fe7-a7e5-acef687b54b3" />

**Comment by Tas4tari** [comment]
On Windows 11, app version 1.5.8, when user is on node: `Local` and switches Tor off, deletes node folder and restarts. The user gets stuck at:
`Connecting network peers 0/3` for `Local` and node configuratuion has jumped to `Remote & Local`

<img width="1636" height="961" alt="Image" src="https://github.com/user-attachments/assets/c45f98bb-6e1a-4d5f-a9a4-c47582f5322a" />

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.13, I find that when I have my configuration set to: 
- `Local`
- Switch Tor off
- Go to Local and delete node folder
- Restart the app

I find that the app is restarted, and config has been switched to: `Remote & Local` with `Local` being stuck on `Connecting to network peers`

<img width="1640" height="960" alt="Image" src="https://github.com/user-attachments/assets/b7b25d64-75cc-485f-b3b8-4bb8ffa4347d" />

**Comment by Tas4tari** [comment]
Tested on Windows 11, app version 1.5.14, I find that when I have my configuration set to:

- Remote & Local
- Switch Tor off
- Go to Local and delete node folder
- Restart the app
I find that the app is restarted, and config has stayed on: `Remote & Local` with `Local` sync being stuck on Connecting to network peers


<img width="1650" height="990" alt="Image" src="https://github.com/user-attachments/assets/1381ad17-692d-4516-9654-24852e3ed5ea" />

**Comment by Tas4tari** [comment]
On Windows 11, app version 1.5.23:
I am able to connect to the TU network, however when I am on "Local" I am stuck on "Connecting to network peers 0/3". I re-enabled Tor and then restarted and was able to continue

**Comment by Tas4tari** [comment]
On Windows 11, app version 1.6.0:
I am able to connect to the TU network, however when I am on "Local" I am stuck on "Connecting to network peers 0/3". I re-enabled Tor and then restarted and was able to continue

**Comment by Tas4tari** [comment]
On Windows 11, app version 1.6.1:
I am able to connect to the TU network, however when I am on "Local" I am stuck on "Connecting to network peers 0/3". I re-enabled Tor and then restarted and was able to continue
